Update direct-routing-sip-specification.md (#119613)
Azure/azure-content: articles/communication-services/concepts/telephony/direct-routing-sip-specification.md

### User-To-User header

> SIP User-To-User (UUI) header is an industry standard to pass contextual information during a call setup process. The maximum length of a UUI header key is 64 chars. The maximum length of UUI header value is 256 chars. The UUI header value may consist of alphanumeric characters and a few selected symbols which includes "=", ";", ".", "!", "%", "*", "_", "+", "~", "-".

Reviewer: ```suggestion
SIP User-To-User (UUI) header is an industry standard to pass contextual information during a call setup process. The maximum length of a UUI header key is 64 chars. The maximum length of UUI header value is 256 chars. The UUI header value might consist of alphanumeric characters and a few selected symbols, including "=", ";", ".", "!", "%", "*", "_", "+", "~", "-".
```

### Custom header

> Azure Communication Services also supports up to five custom SIP headers. Custom SIP header key must start with a mandatory `X-MS-Custom-` prefix. The maximum length of a SIP header key is 64 chars, including the `X-MS-Custom-` prefix. The SIP header key may consist of alphanumeric characters and a few selected symbols which includes ".", "!", "%", "*", "_", "+", "~", "-". The maximum length of SIP header value is 256 chars. The SIP header value may consist of alphanumeric characters and a few selected symbols which includes "=", ";", ".", "!", "%", "*", "_", "+", "~", "-".

Reviewer: ```suggestion
Azure Communication Services also supports up to five custom SIP headers. Custom SIP header key must start with a mandatory `X-MS-Custom-` prefix. The maximum length of a SIP header key is 64 chars, including the `X-MS-Custom-` prefix. The SIP header key might consist of alphanumeric characters and a few selected symbols, including ".", "!", "%", "*", "_", "+", "~", "-". The maximum length of the SIP header value is 256 characters. The SIP header value might consist of alphanumeric characters and a few selected symbols, including "=", ";", ".", "!", "%", "*", "_", "+", "~", "-".
```
